test/commands/modules/disable.spec.ts

```diff
@@ -19,7 +19,10 @@ jest.mock("../../../src/state/state-manager", () => {
     return {
         StateManager: function () {
             return {
-                excludeModule: excludeModuleMock
+                excludeModule: excludeModuleMock,
+                snapshot: {
+                    excludedServices: ["serviceA", "serviceB"]
```

Comment:
Where are you testing a snapshot without `excludedServices`?

Reply:
Still trying to work out how to test that.

Reply:
Fixed in next pr